Sources/AloeStackView/AloeStackView.swift

```diff
@@ -449,6 +449,21 @@ open class AloeStackView: UIScrollView {
       stackView.widthAnchor.constraint(equalTo: widthAnchor)
     ])
   }
+  
+  /// The first row in the stack view.
+  ///
+  /// This is optional because there may not be a row in the stack view.
+  open var firstRow: UIView? {
+    return stackView.arrangedSubviews.first as? StackViewCell
+  }
+  
+  /// The last row in the stack view.
+  ///
+  /// This is optional because there may not be a row in the stack view.
+  open var lastRow: UIView? {
+    return stackView.arrangedSubviews.last as? StackViewCell
+  }
+
```

Comment:
The StackViewCell container is implementation detail and should not be exposed.  These properties should return references to the same view objects that were added (e.g. arrangedSubviews.first?.subviews.first).

Reply:
![Screenshot 2019-04-09 at 19 35 11](https://user-images.githubusercontent.com/31620143/55825850-b4061000-5afe-11e9-9442-10249a4c5299.png)

In my latest commit, I'm retrieving the contentView from the StackViewCell and removing implementation details

Reply:
Oops.  You're right.  How embarrassing.  🙈